DylibBinder/HashSetExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DylibBinder {
+	internal static class HashSetExtensions {
+		public static void AddRange<T> (this HashSet<T> t, params IEnumerable<T> [] entriesArray)
+		{
+			foreach (var entries in entriesArray) {
+				foreach (var entry in entries) {
+					t.Add (entry);
+				}
+			}
```

Comment:
Is this even needed? UnionWith does the same: https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1.unionwith?view=net-5.0

> Modifies the current HashSet<T> object to contain all elements that are present in itself, the specified collection, or both.

Read the docs, usually there is a method that does what you want. Also, params of arrays? You can re-write this with BCL methods probably.